### DISCUSSION TITLE
Possible to install Android on an a Raspberry Pi clone?

### DISCUSSION THREAD

**Discussion by trevorlittle** [tesla-android/issue-tracker#78] [Q&A]
With the shortage/overpriced Raspberry Pi 4s (in the US at least), has any tried to install this project on one of the Raspberry Pi clones like RockPi, OrangePi, BananaPi, etc? I'm wondering if I can get one of those at a fair price if it will work. I have a bunch of Raspberry Pi 3's for the Linux install but nothing better for the Android install.

**Comment by JBaptisme33** [comment]
Same question here :) ?
